MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -222,4 +222,20 @@ + (void)programChangeForPlayer:(nonnull NSString *)name withVideoData:(nullable
     }
 }
 
++ (void)updateCustomerDataForPlayer:(nonnull NSString *)name withPlayerData:(nullable MUXSDKCustomerPlayerData *)playerData withVideoData:(nullable MUXSDKCustomerVideoData *)videoData {
+    MUXSDKPlayerBinding *player = [_viewControllers valueForKey:name];
+    if (player) {
+        if (videoData) {
+            MUXSDKDataEvent *dataEvent = [MUXSDKDataEvent new];
```

Comment:
I'd reorganize this a bit, something like:

```objective-c
if (player) {
  MUXSDKDataEvent *dataEvent = [MUXSDKDataEvent new];
  if (videoData) {
    [dataEvent setCustomerVideoData:videoData];
  }
  if (playerData) {
    [dataEvent setCustomerPlayerData:playerData];
  }
  [MUXSDKCore dispatchEvent:dataEvent forPlayer:name];
}
```

Something like that, but you might want to check to see if it was updated at all. I'd just rather not emit two data events if possible.

Reply:
Oh ya, good call, much more better